inflearn/shopping-mall/client/src/compoents/cart/CartList.tsx

```
@@ -1,17 +1,97 @@
+        checkboxRefs?.forEach((inputElem) => {
+            inputElem.current!.checked = allChecked 
+        })
+    }
+
+    const handleCheckboxChanged = (e?: SyntheticEvent) => {
+        if (!formRef.current) {
+            return;
+        }
+        const targetInput = e?.target as HTMLInputElement
+
+        if (targetInput && targetInput.classList.contains('select-all')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)